### DISCUSSION TITLE
Re-Vanced app has stopped working

### DISCUSSION THREAD

**Discussion by nelisjunior** [ReVanced/revanced-discussions#1109] [Question] [answered]
Today Spotify stopped working. It simply didn't show any songs. I updated the manager and Spotify versions, but it didn't work. 
Now I can't log into my account. When I try to log in, it says an error occurred and the option to try again appears.
My login method is via email and password.
The same happens with my wife's phone.

**Comment by oSumAtrIX** [comment, marked as answer]
Open a bug report in the patches repo
